packages/python-telethon/PKGBUILD

```diff
@@ -4,17 +4,17 @@
 pkgname=python-telethon
 _pkgname=Telethon
 _pyver=3.10
-pkgver=1.13.0
-pkgrel=4
+pkgver=1.24.0
+pkgrel=5
 epoch=1
 pkgdesc='Full-featured Telegram client library for Python 3.'
 arch=('any')
 url='https://pypi.org/project/telethon/#files'
 license=('MIT')
 depends=('python' 'python-pyaes' 'python-rsa')
 makedepends=('python-setuptools')
-source=("https://files.pythonhosted.org/packages/f4/44/547b3f020733724eaf202396f465957972dbaf6a4ef7aeff6fada6bcde28/$_pkgname-$pkgver.tar.gz")
-sha512sums=('5b039be72ede3abeb9a01513416b247af5cc4a37030add6dab2c78a070a88a145524e6f1e6b421b52721fc033e3fd39b331f3dc5933512b36e62598c82a5a42b')
+source=("https://files.pythonhosted.org/packages/71/75/40c3a381b230f7b5b4ea1b48df403590d54001a81e07cf819af812943822/$_pkgname-$pkgver.tar.gz")
```

Comment:
Use this for permaling https://github.com/BlackArch/blackarch-pkgbuilds/blob/4ae2d3f269ad76eda2d78f3ea172e9bba6fd1835/PKGBUILD-python-lib-setuptools#L15

Checking our templates is always a good idea ;)

Reply:
Hope my new commit will resolve this issue.